src/object/endpoint.c

```diff
@@ -108,8 +110,6 @@ void sendIPC(bool_t blocking, bool_t do_call, word_t badge,
         }
         possibleSwitchTo(dest);
 #else
-        bool_t replyCanGrant = thread_state_ptr_get_blockingIPCCanGrant(&dest->tcbState);;
```

Comment:
It looks like the removal of the extra semicolon here is breaking several of the preprocess checks that should pass.

Reply:
Yeah. I think this should be fine for the non-MCS proofs, since the `ccorres_rewrite` machinery should normalise to the same thing.

Reply:
Yes, the proof test succeeded.